python/minmaxmap.py

```diff
@@ -14,9 +16,24 @@
     line = line.strip()
     line = line.split()
     slice_file = os.path.join(line[0])
+    slice = None
 
     #load nifti image
-    slice = nib.load(slice_file)
+    if not os.path.exists(slice_file):
```

Comment:
you should rather check if `slice_file` is a URL with the `hdfs` protocol. That would avoid trying to read the file on HDFS if it doesn't exist. Also, what if the user meant a local file, it doesn't exist and for some reason a file with the same name exist on HDFS?